chatbot/core/aiml/parse_aiml.php

```diff
@@ -212,7 +212,7 @@ function parse_matched_aiml($convoArr, $type = "normal", $aiml_pattern = '')
     //which debug mode?
     runDebug(__FILE__, __FUNCTION__, __LINE__, "Run the aiml parse in $type mode (normal or srai)", 3);
 
-    $aiml_pattern = ($aiml_pattern == '') ? $convoArr['aiml']['pattern'] : $aiml_pattern;
```

Comment:
Ok, fair enough. We can always revisit this if your change cause a problem at some future point. I withdraw my earlier comment.